openshift-knative-operator/pkg/serving/extension.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 const loggingURLTemplate = "https://%s/app/kibana#/discover?_a=(index:.all,query:'kubernetes.labels.serving_knative_dev%%5C%%2FrevisionUID:${REVISION_UID}')"
-const observabilityCMName = "observability"
+const observabilityCMName = "config-observability"
```

Comment:
This is not correct. While the CM is indeed named `config-observability`, the `config-` prefix is swallowed in the CR.

Reply:
Yeah I will update it.